From a54917fdffef8fc35b2d827a24ced41eeec575bb Mon Sep 17 00:00:00 2001
From: Jordan Petridis <jpetridis@gnome.org>
Date: Mon, 22 Sep 2025 15:09:35 +0300
Subject: [PATCH] linux: Enable SIMPLEDRM driver

Along with SYSFB_SIMPLEFB for UEFI

This lets us work with buffers initialized by the
firmware before we booted. Plymouth can make use of
SIMPLEDRM

(cherry picked from commit 6debfcee1ac0e02785cb0d9cfa86a6ffc5e0dafd)

Co-authored-by: Jordan Petridis <jpetridis@gnome.org>
---
 files/linux/fdsdk-config.sh | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/files/linux/fdsdk-config.sh b/files/linux/fdsdk-config.sh
index 8247f11a4b..0958a5ebcb 100644
--- a/files/linux/fdsdk-config.sh
+++ b/files/linux/fdsdk-config.sh
@@ -643,20 +643,22 @@ module NIU
 module SUNGEM
 module TEHUTI
 module TLAN
 module VIA_RHINE
 module VIA_VELOCITY
 if has PCMCIA; then
     module PCMCIA_XIRC2PS
 fi
 
 # Common DRM drivers
+enable DRM_SIMPLEDRM
+enable SYSFB_SIMPLEFB
 module DRM_NOUVEAU
 module DRM_RADEON
 module DRM_AMDGPU
 enable DRM_AMDGPU_SI
 enable DRM_AMDGPU_CIK
 enable DRM_AMD_DC
 case "$arch" in
     x86_64)
         enable DRM_AMD_DC_FP
         enable DRM_AMD_SECURE_DISPLAY
-- 
2.50.1

